lms/djangoapps/courseware/views/views.py

```diff
@@ -1637,6 +1637,7 @@ def get_financial_aid_courses(user):
     for enrollment in CourseEnrollment.enrollments_for_user(user).order_by('-created'):
 
         if enrollment.mode != CourseMode.VERIFIED and \
+                enrollment.course_overview and \
```

Comment:
@rlucioni The examples I found about a course without course_overview is an MIT course from 2014. So those are very old course, and likely having problem loading the course from module store.